### ISSUE TITLE
Numbers of contaminants appear small in mock QA

### ISSUE THREAD

**Issue by geordie666** [desihub/desitarget#267]
For example, the ELG `True Redshift` plot at:

http://portal.nersc.gov/project/cosmo/temp/ioannis/refactor/ELG.html

Only appears to have < 5% stellar contamination. Is this realistic?

**Comment by moustakas** [comment]
Right, the density of contaminants still needs to be tuned.  (I'll spin this off into a dedicated issue once #264 is merged).

**Comment by moustakas** [comment]
First pass of contaminants was addressed in #427.  I'll open a separate issue for tuning this model further.
